notebooks/tel_and_site/subsys_req_ver/m2/SITCOMTN-096_M2_outer_control_closed_loop.ipynb

```diff
@@ -0,0 +1,1318 @@
+{
```

Comment:
<p>I suggest to avoid redundancy, you could put the timestamp directly in the following strings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Why I put this within the different cell and variable is in case when the notebook is used for the future tests. Even I guess that this will not happened for M2 outer-loop test, but I want to put it on the different cell to allow users to change the timestamp easily. </p>

Reply:
Ok Yijung fine with me, leave that block as you had initially. Thanks